diff --git a/src/clam_moveit_config/launch/move_group.launch b/src/clam_moveit_config/launch/move_group.launch
index 930baf3..8902412 100644
--- a/src/clam_moveit_config/launch/move_group.launch
+++ b/src/clam_moveit_config/launch/move_group.launch
@@ -11,7 +11,7 @@
 
   <arg name="allow_trajectory_execution" default="true"/>
   <arg name="max_safe_path_cost" default="1"/>
-  <arg name="publish_monitored_planning_scene" default="false"/>
+  <arg name="publish_monitored_planning_scene" default="true"/>
 
   <include ns="move_group" file="$(find clam_moveit_config)/launch/planning_pipeline.launch">
     <arg name="pipeline" value="ompl" />
diff --git a/src/clam_moveit_config/launch/moveit_rviz.launch b/src/clam_moveit_config/launch/moveit_rviz.launch
index 07e6d90..1308d23 100644
--- a/src/clam_moveit_config/launch/moveit_rviz.launch
+++ b/src/clam_moveit_config/launch/moveit_rviz.launch
@@ -4,7 +4,8 @@
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
   <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
   
-  <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false" output="screen">
+  <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" 
+	respawn="false" output="screen" args="-d $(find clam_moveit_config)/launch/moveit.rviz" >
     <rosparam command="load" file="$(find clam_moveit_config)/config/kinematics.yaml"/>
   </node>
 
